ui/src/lib/access.worker.js

```diff
@@ -3,14 +3,22 @@ import "./wasm_exec.js";
 
 const go = new Go();
 
-const wasmInit = WebAssembly.instantiateStreaming(
-	fetch("/access.wasm"),
-	go.importObject
-);
+const instantiateStreaming =
+	WebAssembly.instantiateStreaming ||
+	async function (resp, importObject) {
+		const response = await resp;
+		const source = await response.arrayBuffer();
 
-wasmInit.then((result) => {
-	go.run(result.instance);
-});
+		return await WebAssembly.instantiate(source, importObject);
+	};
+
+const response = fetch("/access.wasm");
+instantiateStreaming(response, go.importObject)
+	.then((result) => {
+		go.run(result.instance);
+		self.postMessage("configured");
```

Comment:
No need to post messages. This is for their worker code and not needed for ours.